<launch>
    <arg name="namespace" default="uav0"/>
    <arg name="offset_x" default="0.0"/>
    <arg name="offset_y" default="0.0"/>
    <arg name="init_x" default="0.0"/>
    <arg name="init_y" default="0.0"/>
    <arg name="init_z" default="10.0"/>

    <group ns="$(arg namespace)">
       <include file="$(find utm)/launch/offboard_launch.launch">
            <!--offsets from airsim look at json file settings-->
            <arg name="offset_x" value="$(arg offset_x)"/>
            <arg name="offset_y" value="$(arg offset_y)"/>

            <arg name="init_x" value="$(arg init_x)"/>
            <arg name="init_y" value="$(arg init_y)"/>
            <arg name="init_z" value="$(arg init_z)"/>
        </include> 
    </group>
 
</launch>